e2e/test-component/scenarios/embedding-sdk/interactive-dashboard.cy.spec.tsx

```diff
@@ -20,7 +20,7 @@ describeEE("scenarios > embedding-sdk > interactive-dashboard", () => {
   beforeEach(() => {
     signInAsAdminAndEnableEmbeddingSdk();
 
-    const textCard = getTextCardDetails({ col: 16, text: "Text text card" });
+    const textCard = getTextCardDetails({ col: 16, text: "Test text card" });
```

Comment:
I believe this is a typo, since there's a counterpart `Test question card` in StaticDashboard test suite.